admin/tabs/general/roles/details/members/RoleMembersModel.ts

```diff
@@ -175,13 +187,39 @@ export class RoleMembersModel extends HoistModel {
                 {
                     field: 'name',
                     autosizeMaxWidth: 300,
-                    renderer: (name, {record}) =>
-                        record.get('type') === types.DIRECTORY_GROUP
-                            ? this.fmtDirectoryGroup(name)
-                            : name,
-                    tooltip: true
+                    renderer: (name, {record}) => {
```

Comment:
might help put this grid config on a diet if we were to define this renderer and the renderer below as functions at the bottom of the file.